Automattic/wp-calypso
# Theme Discovery: Remove Discovery view and default back to Recommended view

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4442

## Proposed Changes

* We've removed the Discovery view and reverted to the Recommended View.
* The collection modifier still works but isn't accessible until we implement the Collection Carousel in the Recommended view, which will be done in a different issue.

## Testing Instructions



* Open the Calypso Live link
* Go to the LoTS & LiTS and use the corresponding flag `flags=themes/discovery-{lots|lits}`
* You should only see the `Recommended` tab and view.
* There should be no references to the `Discover` tab and view.
* Open a compatible category and add `/collection` to the end of the URL, you shall see the corresponding collection view.

Before: 
![Captura de pantalla 2023-11-02 a las 11 01 50](https://github.com/Automattic/wp-calypso/assets/1989914/ca383034-fc92-4bf8-a1bd-83052b603ba8)

After:
![Captura de pantalla 2023-11-02 a las 11 03 08](https://github.com/Automattic/wp-calypso/assets/1989914/d74e99d8-0b58-44cd-bb3f-a50b5ed397f9)


## Pre-merge Checklist
